continuousflex/protocols/pdb/protocol_pseudoatoms_base.py

```diff
@@ -106,17 +107,17 @@ def convertToPseudoAtomsStep(self, inputFn, fnMask, sampling, prefix=''):
                   "-v 2 --intensityColumn Bfactor"
         if fnMask:
             params += " --mask binary_file %(fnMask)s"
-        self.runJob("xmipp_volume_to_pseudoatoms", params % locals())
+        self.runJob("xmipp_volume_to_pseudoatoms", params % locals(), env=Domain.importFromPlugin('xmipp3').Plugin.getEnviron())
```

Comment:
Do we need to add this now each time we call a Xmipp program ?

Reply:
Actually it depends how you call the program. If you use self.runJob() then yes. There is a function that can be used without passing this environment called runProgram() 
https://github.com/scipion-em/scipion-em-continuousflex/blob/master/continuousflex/protocols/protocol_tomoflow.py#L36
However, when there is an mpi job (which goes and uses the binary that starts with mpi_) , runJob does it automatically. Here I could use runProgram